src/components/Explore/FavoriteButton.tsx

```diff
@@ -1,37 +1,45 @@
 import useTheme from 'hooks/useTheme'
 import { useAtom } from 'jotai'
-import { showFavoritesAtom } from 'pages/Explore/index'
+import { darken } from 'polished'
 import { Heart } from 'react-feather'
 import styled from 'styled-components/macro'
 
+import { showFavoritesAtom } from './TokenTable'
+
 const FavoriteButtonContent = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
   gap: 8px;
 `
-const StyledFavoriteButton = styled.button`
+const StyledFavoriteButton = styled.button<{ active: boolean }>`
   padding: 0px 16px;
   border-radius: 12px;
   background-color: ${({ theme }) => theme.bg0};
-  border: 1px solid ${({ theme }) => theme.bg0};
-  color: ${({ theme }) => theme.text1};
+  border: 1px solid ${({ theme, active }) => (active ? '#869EFF' : theme.bg1)};
```

Comment:
could you add this color to the theme file and use it as a constant from there? https://github.com/Uniswap/interface/blob/main/src/theme/index.tsx